blocks/gmo-program-list/gmo-program-list.js

```diff
@@ -126,11 +128,10 @@ export default async function decorate(block, numPerPage = currentNumberPerPage,
 }
 
 function debug_console(){
```

Comment:
Remove this function as this is not being called and remove line number 127 code(`//debug_console();`) as well